src/design-tokens/tier-1-definitions/layout.json

```diff
@@ -4,6 +4,9 @@
       "max-width": {
         "value": "71.25rem"
       },
+      "sidebar-width": {
+        "value": "13.4375rem"
```

Comment:
Could we round up to `13.5rem` instead? (it's a difference of 1 pixel, from 215 -> 216px)